packages/react-mutation-mapper/src/component/oncokb/OncoKbCardTreatmentContent.tsx

```diff
@@ -0,0 +1,280 @@
+import { DefaultTooltip } from 'cbioportal-frontend-commons';
+import { Citations, IndicatorQueryTreatment } from 'oncokb-ts-api-client';
+import classnames from 'classnames';
+import { action, observable } from 'mobx';
+import { observer } from 'mobx-react';
+import * as React from 'react';
+
+import { ICache } from '../../model/SimpleCache';
+import OncoKbTreatmentTable from './OncoKbTreatmentTable';
+import ReferenceList from './ReferenceList';
+import SummaryWithRefs from './SummaryWithRefs';
+
+import mainStyles from './main.module.scss';
+import tabsStyles from './tabs.module.scss';
+import autobind from 'autobind-decorator';
+
+type OncoKbCardTreatmentContentProps = {
+    variant: string;
+    oncogenicity: string;
+    mutationEffect: string;
+    mutationEffectCitations: Citations;
+    geneSummary: string;
+    variantSummary: string;
+    tumorTypeSummary: string;
+    biologicalSummary: string;
+    treatments: IndicatorQueryTreatment[];
+    pmidData: ICache;
+    usingPublicOncoKbInstance: boolean;
+};
+
+const ONCOKB_DATA_ACCESS_PAGE_LINK =
+    'https://docs.cbioportal.org/2.4-integration-with-other-webservices/oncokb-data-access';
```

Comment:
i see a few urls embedded in these views.  Can we forsee these needing instance specific configuration?  is it worth trying to externalize them to urls.ts?

Reply:
It's a general guidance.